datastruc_ex7.py

```diff
@@ -63,15 +63,13 @@ def dataFunction(n: int, rand: bool, lst: list):
     dictofreq = freqDict(fromListToString(wordSelected(n=n,
                                                        random=rand,
                                                        lst=lst)))
-    max_val = maxCharacter(dictofreq)
-    median_val = findMedian(dictofreq)
-    #print(f'The frequency dictionary is: {dictofreq}\
-    #\nThe most common character is: {max_val}\
-    #\nThe median value is: {median_val}')
-    print(
-        f'''The frequency dictionary is: {dictofreq}
-        The most common character is: {max_val}
-        The median value is: {median_val}''')
+    print(f'The frequency dictionary is: {dictofreq}\
+    \nThe most common character is: {maxCharacter(dictofreq)}\
+    \nThe median value is: {findMedian(dictofreq)}')
+    #print(
+        #f'''The frequency dictionary is: {dictofreq}
+        #The most common character is: {maxCharacter(dictofreq)}
+        #The median value is: {findMedian(dictofreq)}''')
```

Comment:
Perfect, See how the first function is too long for the print, so we use a variable, but the other two are compact and short so we can make the code more clean.